docker-for-mac/index.md

```diff
@@ -462,6 +462,28 @@ ln -s $etc/docker.zsh-completion /usr/local/share/zsh/site-functions/_docker
 ln -s $etc/docker-compose.zsh-completion /usr/local/share/zsh/site-functions/_docker-compose
 ```
 
+### Fish-Shell
+
+Fish-shell also supports tab completion [completion
+system](https://fishshell.com/docs/current/#tab-completion){:target="_blank"
+class="_"}. To activate completion for Docker commands,
+these files need to be copied or symlinked to your Fish-shell `completions/`
+directory.
+
+Create at first the `completions` directory if not exist:
+
+```bash
+mkdir ~/.config/fish/completions
```

Comment:
Can you add `-p`, so that it will create the directory if it doesn't exist, but doesn't produce an error if it already did?

```suggestion
mkdir -p ~/.config/fish/completions
```